lib/sdk/writer/block-stream.js

```diff
@@ -16,11 +16,13 @@
 
 'use strict'
 
+const directIo = require('@ronomon/direct-io')
 const stream = require('readable-stream')
 const debug = require('debug')('etcher:writer:block-stream')
 
 const MIN_BLOCK_SIZE = 512
 const CHUNK_SIZE = 64 * 1024
+const DIRECT_IO_ALIGNMENT = 4096
```

Comment:
This should be defined in one place only. (It was already defined above in block-read-stream.js).